internal/plugin/connectors/http/generic/config.go

```diff
@@ -15,6 +15,7 @@ import (
 type config struct {
 	CredentialPatterns map[string]*regexp.Regexp
 	Headers            map[string]*template.Template
+	OAuth1             map[string]*template.Template
```

Comment:
This needs a touch better name. what is `Oauth1`? Object? Auth info? Auth data? etc

Reply:
`OAuth1Secrets`?

Reply:
It's for the 4 secrets passed in in the config